common/src/cams/validation.test.ts

```diff
@@ -231,6 +231,14 @@ describe('validation', () => {
         reasons: ['Value must be an object'],
       });
     });
+
+    test('should return VALID for falsy spec values', () => {
```

Comment:
I don't understand the title. Why is it "should return VALID..." when it's returning reasons for why it is not valid?